pin-devices-to-current-release.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+## This script sets devices running a commit to be pinned to that specific commit
+## Usage: ./pin-devices-to-current-release <FULL_COMMIT_HASH>
```

Comment:
The name of this file is a bit weird, I would assume that `pin-devices-to-current-release` would take the commit value from the application, not the command line.

Reply:
Good point, changed it to use @thgreasi's suggestion